### PR TITLE
Laravel 11 support

### PR DESCRIPTION
**Pull request by asterd**
Enable laravel 11 support

### CODE REVIEW

**Comment by jsh-avrillo** [comment]
@TishoTM @d13r - Can this be merged please?

**Comment by d13r** [comment]
> @TishoTM @d13r - Can this be merged please?

I'm not a project collaborator, so don't have that ability! I'm only listed under "Contributors" because of #6.

File: composer.json [hunk 1]
```diff
@@ -2,10 +2,10 @@
     "name": "tishotm/eloquent-ci-relations",
     "description": "Eloquent case-insensitive relations",
     "require": {
-        "illuminate/database": "^5.5.33 || ^6.20.26 || ^7.30.4 || ^8.40.0 || ^9.0 || ^10.0"
+        "illuminate/database": "^5.5.33 || ^6.20.26 || ^7.30.4 || ^8.40.0 || ^9.0 || ^10.0 || ^11.0"
     },
     "require-dev": {
-        "php": "~7.1 || ~8.0",
+        "php": "~7.1 || ~8.0 || ~8.2",
```
Comment (TishoTM) [inline composer.json:8]: declaring `~8.2` is not necessary

**Comment by TishoTM** [comment]
The unit tests are broken in PHP 8.2. 

@asterd, could you add `protected $db;` to the `TishoTM\Tests\Eloquent\TestBase` class.
... and remove the `|| ~8.2` from the composer.json

**Comment by asterd** [comment]
Ok done @TishoTM

**Comment by TishoTM** [comment]
Thanks! I will merge it later today

**Pull request merged**